### DISCUSSION TITLE
How Automatic Hosts and SSL Generate.

### DISCUSSION THREAD

**Discussion by pilotrick** [ej52/proxmox-scripts#58] [Q&A]
Hi, I have 3 CT with dockers on each one, so I want a soluctions especial with a wildcard domains I can also pay for that or sponsor this for 4 months. 

So I have an wildcard domains .*examples.com and 1 of the CT have a docker just for this domains, so any time a new container is created it exposes obsiviously a new port  when I create newsub.examples.com on port 8099 so I want that npm can create a hots and ssl  , I don't know if there's a new alternative for this.

Please Let Know

**Comment by jeffryjdelarosa** [comment]
I want this solutions Also.

**Reply by ej52** [reply]
@pilotrick NginxProxyManager has no automated host creation, your best bet if using docker would be [traefik](https://doc.traefik.io/traefik/) or [nginx-proxy](https://github.com/nginx-proxy/nginx-proxy)

**Reply by jeffryjdelarosa** [reply]
Hi, could you help me with this or if there's a documentation about it.
